uPortal-webapp/src/main/webapp/media/skins/respondr/view/less/activity.less

```diff
@@ -0,0 +1,63 @@
+/**
```

Comment:
That's reasonable.

The "common" in `/skins/respondr/common/` was chosen to signify that things inside that folder are "common" to all Respondr skins.  All the other folders under `/skins/respondr/` (besides `common/`) are actual skins... e.g. `defaultSkin`, `myUSD`, `scccd`, _etc_.  This is a pattern that continues from Universality.